node/src/command.rs

```diff
@@ -603,7 +603,6 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 		"darwinia" => Box::new(darwinia_chain_spec::config()),
 		"darwinia-genesis" => Box::new(darwinia_chain_spec::genesis_config()),
 		"darwinia-dev" => Box::new(darwinia_chain_spec::development_config()),
-		"darwinia-local" => Box::new(darwinia_chain_spec::local_testnet_config()),
```

Comment:
I think we can keep it, `dev` is used to run a testnet that does not require a separate relay chain (I will make it work in another PR), and `local` is used to run a testnet that requires a separate relay chain.